earthly/postgresql/Earthfile

```diff
@@ -34,8 +37,10 @@ postgres-base:
         ripgrep \
         wget \
         pipx \
-        && rm -rf /var/lib/apt/lists/*
-
+        && rm -rf /var/lib/apt/lists/* \
+        && wget -O /bin/schemaspy.jar https://github.com/schemaspy/schemaspy/releases/download/v${SCHEMASPY_VERSION}/schemaspy-${SCHEMASPY_VERSION}.jar \
```

Comment:
I would be inclined to put any `wget` in its own `RUN`.  Otherwise, a change in any of these versions will require all the dependencies being `apt-get` installed to be re-installed, when they are not needed to be re-installed.

If the JDBC_VERSION is in your opinion, less likely to change than the SCHEMASPY_VERSION it should also be `wgetted` first, so that its not necessary to pull it again if SCHEMASPY_VERSION changes, but it does not.

As soon as a RUN can not be cached, then every other command after it will no longer be cached.  This is one of the motivations of putting things in their own targets, to avoid unnecessary rebuilds/installs of dependencies.
So the ordering should be from "least likely to change"  to "most likely".  Some of this is subjective, so there is no strict "right" answer to the dependency install order, but it is something to consider.

By the same token, it might be worth moving the `java+COPY_DEPS` after the `apt-get` but before the wgets.

I would also consider putting the wgets after the sqlfluff install for the same reason, we change its version rarely.